### ISSUE TITLE
Autorun doesn't work.

### ISSUE THREAD

**Issue by paulscottrobson** [commanderx16/x16-rom#130]
Attached prg file (autoloads at $1000) started using -scale 2 -prg kernel.prg -run -dump R -debug runs on r36, on current displays BASIC prompt and requires SYS4096 to start (Arch Linux)

[kernel.zip](https://github.com/commanderx16/x16-rom/files/4049089/kernel.zip)

**Comment by jestin** [comment]
I've been seeing this as well.  Again, the r36 tag of the rom works fine.  It does not seem to matter which version of the emulator is being used (I've been using the HEAD of master, but I get the same results on tag r36).  Running the program manually with `SYS4096` or `SYS$1000` works fine.  My run command is `x16emu -prg myapp.prg -run -scale 2`

**Comment by jestin** [comment]
I thought I was seeing the same thing, but I realized that it was the emulator version, not the ROM version, that needed to be checked out at the latest tag.  I am able to get autorun to work again by using the r36 tag for the emulator, and the HEAD of master for the rom (commit 31a9470 at the time of this posting).  I suspect that this is an emulator issue, as I can't seem to get autorun to work no matter which ROM I have checked out.
